marketplace/config/settings/configurator.py

```diff
@@ -0,0 +1,214 @@
+#
+# Copyright 2020 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+"""
+Handler module for gathering configuration data.
+"""
+from .env import ENVIRONMENT
+
+
+CLOWDER_ENABLED = ENVIRONMENT.bool("CLOWDER_ENABLED", default=False)
+if CLOWDER_ENABLED:
+    from app_common_python import LoadedConfig, KafkaTopics, ObjectBuckets, SmartAppConfig
+
+class Configurator:
+    """Obtain configuration based on mode."""
+
+    @staticmethod
+    def get_kafka_broker_host():
+        """Obtain kafka broker host address."""
+        if CLOWDER_ENABLED:
```

Comment:
This is not really a problem from an integration perspective, but imho I'd implement two subclasses of Configurator, one for clowder_enabled=true and another for clowder_enabled=false and return one or the other depending on the value of clowder_enabled.

Reply:
make sense. made that change.